app/models/maintenance_tasks/run.rb

```diff
@@ -206,6 +208,26 @@ def csv_attachment_presence
       nil
     end
 
+    # Performs validation on the arguments to use for the Task. If the Task is
+    # invalid, the errors are added to the Run.
+    def task_arguments_are_valid
```

Comment:
This method is saying that the argument are valid, but what it does is check if the arguments are valid. SInce it is a public method that people can call I still think it should be called `validate_task_arguments`. That is the action being done by the method. Or `check_if_ task_arguments_are_valid`

Reply:
Isn't it redundant to repeat `validate` or to prepend with `check_if_`? I was styling it how [Rails says to write custom validations](https://guides.rubyonrails.org/active_record_validations.html#custom-methods):
```ruby
class Invoice < ApplicationRecord
  validate :active_customer, on: :create

  def active_customer
    errors.add(:customer_id, "is not active") unless customer.active?
  end
end
```

ie. `validate <condition_that_should_be_met_for_record_to_be_valid>`

Reply:
It is, but those methods are usually private and you don't expect users to call `invoice.active_customer`. Our method here is public though, and it is being called in the test, so its usage, is different.

Reply:
Gotcha, makes sense. Will update 👍